invenio/modules/pages/admin.py

```diff
@@ -60,6 +62,7 @@ class PagesAdmin(ModelView):
 
     form_args = dict(
         template_name=dict(
+            default="default.html",
```

Comment:
Shouldn't we go here with sth like `PAGES_DEFAULT_TEMPLATE` from `pages/config.py`